gallerypage.py

```
@@ -48,6 +56,11 @@ def reset_usage():
     for category in KEYWORD_USAGE:
         for regex in KEYWORD_USAGE[category]:
             KEYWORD_USAGE[category][regex] = []
+            
+class FixIndent(yaml.Dumper):
+
+    def increase_indent(self, flow=False, indentless=False):
```

Ruff (0.14.14)

Boolean default positional argument in function definition

(FBT002)